build.gradle.kts

```diff
@@ -21,6 +21,15 @@ plugins {
     id(Config.ApplyPlugins.KT_LINT) version Config.KTLINT_GRADLE_VERSION
     id(Config.ApplyPlugins.DETEKT) version Config.DETEKT_VERSION
     id(Config.ApplyPlugins.KSP) version Config.KSP_VERSION
+    // https://developers.google.com/maps/documentation/android-sdk/secrets-gradle-plugin
+    id(Config.ApplyPlugins.GOOGLE_MAPS_SECRETS_PLUGIN) version Config.MAPS_SECRETS_VERSION
```

Comment:
I like that there is a plugin just for google maps secrets now.

Reply:
TIL!

Just note that you'll need to store a secret `local.properties` file with the maps api key on your CI platform (differs per vendor) and bring the file into the workspace prior to running the gradle tasks to compile the project. If your maps api key differs per variant (not sure how likely this is), then you'll also need to account for that additional logic in your CI platform build.